test/hashmaps/test_maps.fypp

```diff
@@ -24,11 +24,13 @@ module test_stdlib_chaining_maps
     integer, parameter        :: test_size = rand_size*4
     integer, parameter        :: test_16 = 2**4
     integer, parameter        :: test_256 = 2**8
-    ! key_type = 2 to support int8 and int32 key types tested.  Can be 
+    ! key_type = 5 to support int8 and int32 key types tested.  Can be 
     ! increased to generate additional unique int8 vectors additional key types.   
-    integer, parameter        :: key_types = 2
+    integer, parameter        :: key_types = 5
+    character(len=16)         :: char_type 
```

Comment:
The suffix `_type` is usually reserved/used for DT in `stdlib`. Could `char_size` or `char_bytes` be an alternative?

Reply:
This has been changed to char_size